student-cvs/diirfv-7677/cv.html

```diff
@@ -49,7 +51,7 @@ <h1>Formación complementaria</h1>
         <section>
             <h1>Idiomas</h1>
             <ul>
-                <li>Italiano <strong>A2 en curso</strong></li>
+                <li>Italiano <strong>A2</strong><em> en curso</em></li>
```

Comment:
I'd like to see your websites in spanish, english and italian :O !!

Reply:
Ha ha ha, ok, in the future I will do it when we learn how to change the language depends of the origin of the user